entries/iteration.qmd

```diff
@@ -10,20 +10,26 @@ of time-boxed periods ("iterations") of work that follow a specific pattern and
 for incrementally building or improving things. An iteration for us follows
 this general sequence of steps:
 
-- Team lead creates a GitHub Project Board for the upcoming iteration using the ["Iteration" template](https://github.com/orgs/seedcase-project/projects/4).
-    - Team lead will add the aim and output to be discussed and decided on in the first meeting.
-- The first meeting of the iteration, which we aim to be no more than an hour, is to discuss and plan the specifics of the iteration.
-    - Before the first meeting, review the aim and output of the iteration board and then write out potential tasks that we would need to do to complete the output.
-    - During the meeting, decide on the end output (the "increment" or "milestone").
-    - During the meeting, brainstorm and write out more tasks that are needed to complete the increment or milestone, or select them from the existing list of tasks. A task is a GitHub Issue.
-    - Each team member self-assigns themselves to a task (self-assign to Issue) and estimates the time spent on them by using the Iteration's Roadmap Board.
-    - Decide on how long the iteration should or will take and set a date for the "last iteration meeting".
-- Throughout the iteration, we will:
+1. Team lead sets everything up for the next two or three rounds.
+    - Will create a GitHub Project Board for the upcoming iteration using the ["Iteration" template](https://github.com/orgs/seedcase-project/projects/4).
+    - Will add the aim, output, as well as the end date of the iteration to be discussed and decided on in the first meeting.
+    - Will start adding some tasks into the board.
+2. The first planning meeting for the iteration is scheduled well in advance, for between 1-2 hours and right after the previous iteration's debrief. Preferably this meeting is in person.
+    - The aim of the meeting to discuss and plan the specifics of the iteration.
+    - Before the planning meeting, team members will review the aim and output of the iteration board and write out potential tasks (as GitHub Issues) needed to complete the aim.
+    - During the meeting, we'll discuss and decide on the end output (the "increment" or "milestone").
+    - During the meeting, we'll brainstorm and write out more tasks that are needed to complete the increment or milestone, or select them from the existing list of tasks. 
+    - Each team member self-assigns themselves to a task (self-assign to Issue) and estimates the time spent on them by using the Iteration's Roadmap Board. 
+    - We'll review whether the number of tasks, time estimation of those tasks, and the aim of the iteration are realistic in the time available.
```

Comment:
I'm not sure if I see that value of adding that. I would assume things that impact time to complete a task (how much learning is required, sick leave, vacation, caretaking duties, etc) all contribute to those estimates, right?

Reply:
Fair enough. I was thinking that, for instance, when I was preparing with the AdvancedR course, it took up a lot of my time. Or, in January I will need to take some time to start developing the course material for the RPackage course. This will not affect how much time it will take to complete a task, but it does affect how much time I have available to work on Sprout

Reply:
@lwjohnst86 I agree with @signekb, it might be good to make it explicit that there will be iterations where a member of the team will have to focus some of their time on other projects.  Just to make it clear to people joining in the future that this is how we operate.

Reply:
I don't think it is necessary to include these "caveats" in this item, because all of those should be factored into when you estimate time spent on tasks. However, I have added a general caveat to the introduction which I hope clarifies things for both of you :relaxed: Let me know if that helps :relaxed:

Reply:
Alright. It's clearer, but I would still argue that this wouldn't be explicitly included in time spent on the individual tasks. 
I think that it's relevant, during the iteration planning meeting, to talk about what other tasks outside of Seedcase that we will need to spend time on during an iteration ☺️